aws/nextflow-security.tf

```diff
@@ -0,0 +1,19 @@
+resource "aws_security_group" "nf_security" {
+  name = "nextflow-security-group"
+
+  egress {
+    from_port   = 0
+    to_port     = 0
+    protocol    = "-1"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+}
+
+resource "aws_vpc" "nf_vpc" {
+  cidr_block = "10.1.0.0/16"
+}
+
+resource "aws_subnet" "nf_subnet" {
+  vpc_id = aws_vpc.nf_vpc.id
+  cidr_block = "10.1.1.0/24"
```

Comment:
Are you planning on adding additional subnets? If not, you may just want to have this match the VPC's CIDR block.

Reply:
I don't expect so? This is where I was completely lost, and really just was going off examples. I don't know what the function of multiple subnets would be, but I needed to make at least one, it seemed. the CloudFormation-created vpc definitely made more than one, but I don't know why, or how that might be used.

Reply:
Ok then this is should be fine. It shouldn't really matter much anyway.